core/src/main/scala/cats/data/Kleisli.scala

```diff
@@ -68,6 +68,9 @@ final case class Kleisli[F[_], A, B](run: A => F[B]) { self =>
   def tapWith[C](f: (A, B) => C)(implicit F: Functor[F]): Kleisli[F, A, C] =
     Kleisli(a => F.map(run(a))(b => f(a, b)))
 
+  def toReader: Kleisli[Id, A, F[B]] =
+    mapF[Id, F[B]](scala.Predef.identity)
```

Comment:
at least we can save the identity call right? 
```scala
def toReader: Kleisli[Id, A, F[B]] = Kleisli[Id, A, F[B]](run)
```

Reply:
Good point! Updated.